python/perf-kernels/flash-attention.py

```diff
@@ -1102,7 +1082,9 @@ def varlen_input_helper(Z, HQ, HK, N_CTX_Q, N_CTX_K, D_HEAD, dtype):
 @pytest.mark.parametrize('use_alibi', [True, False])
 def test_op_fwd(Z, HQ, HK, N_CTX_Q, N_CTX_K, D_HEAD, causal, use_alibi, dtype=torch.float16):
     torch.manual_seed(20)
-    q, k, v, input_metadata = input_helper(Z, HQ, HK, N_CTX_Q, N_CTX_K, D_HEAD, dtype)
+    # TODO: Adapt test for bshd
+    layout = 'bhsd'
```

Comment:
Do we need to add the layouts quickly for some reason? It would be nice if we could test at least one other layout. If we need to move quick, I am fine adding the tests later.

Reply:
No, not necessary to get it in quickly. I guess I was just being lazy. Good to have tests - Done.